sec.club/src/components/EventBanner/EventBanner.test.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { render, unmountComponentAtNode } from "react-dom"
+import { mount } from 'enzyme'
+import "@testing-library/jest-dom"
+
+import EventBanner from "./EventBanner.jsx"
+
+let container = null;
+describe("Event container appearance", () => {
+    beforeEach(() => {
+        container = document.createElement("div")
+    })
+
+    afterEach(() => {
+        unmountComponentAtNode(container)
+    })
+
+    it("should render without crashing", () => {
+        render(<EventBanner flyerSource="LearnHack.png"/>, container)
+    })
+
+    it("should contain the text content", () => {
+        let wrapper = mount(<EventBanner title="Title" desc="Desc" flyerSource="LearnHack.png"/>, container)
+        expect(wrapper.find(".title").text()).toEqual("Title");
+        expect(wrapper.find(".description").text()).toEqual("Desc");
+    })
+
+    it("should contain the image content", () => {
+        let wrapper = mount(<EventBanner flyerSource="LearnHack.png"/>)
+        let image = require(`../../assets/flyers/LearnHack.png`).default
+        expect(wrapper.contains(<img src={image} alt="LearnHack.png"/>)).toEqual(true)
```

Comment:
This becomes brittle if we decide to add/change the order of additional attributes on `img`.
I would change this to check for the `src` attribute.